thoth/storages/graph/postgres.py

```diff
@@ -73,9 +76,11 @@
 from .models import ExternalSoftwareEnvironment
 from .models import HardwareInformation
 from .models import InspectionRun
+from .models import KebechetGithubAppInstallations
 from .models import PackageExtractRun
 from .models import ProvenanceCheckerRun
 from .models import PythonArtifact
+from .models import PythonPackageVersionEntityRulesAssociation
```

Comment:
i see all the package have kept alphabetically ordered, this should go below.

Reply:
Thanks. Adjusted in a9d8629